package.json

```diff
@@ -46,6 +46,7 @@
     "commander": "2.9.0",
     "event-stream": "3.3.4",
     "express": "4.15.2",
+    "express-handlebars": "^3.0.0",
```

Comment:
Like, do you mean run webpack on the server, or do you mean just render the index on the client.

If you mean server, then no, let's not do that, that's horrible.

If you mean on the client, we could, but it would increase our initial render time because currently we render it on the server before passing it back, if we passed back the index.html (which will be smaller soon) and all the other templates, plus the data for the client to render, before we can do any connections to the socket etc, that will increase it quite a bit. So, I'd really rather not. It's just 1 extra dependency and doesn't seem that big a deal.